cmd/build/bootstrap.go

```
@@ -55,8 +67,119 @@ func bootstrap(ctx context.Context) error {
+`, debugDir, debugDir, debugDir, debugDir, debugDir, debugDir)
+	return kubectl.Bash(ctx, script)
+}
+
+func printSavedBootstrapJobLogs() {
+	path := filepath.Join(bootstrapDebugDir(), "job-logs.txt")
+	data, err := os.ReadFile(path)
+	if err != nil || len(data) == 0 {
+		fmt.Fprintf(os.Stderr, "no captured bootstrap job logs at %s\n", path)
+		return
+	}
+	fmt.Fprintf(os.Stderr, "\n===== captured bootstrap job logs =====\n%s\n===== end captured bootstrap job logs =====\n", data)
```

golangci-lint (2.12.2)

[error] The line is 125 characters long, which exceeds the maximum of 120 characters.

(lll)